spec/spec_helper.rb

```diff
@@ -14,9 +15,13 @@
 end
 
 def fixture_dir
-  File.expand_path "./fixtures", File.dirname(__FILE__)
+  File.expand_path "./fixtures", __dir__
```

Comment:
you don't need to prefix with `./` here:
`  File.expand_path "fixtures", __dir__`